L1Trigger/L1THGCal/src/backend/HGCalMulticlusteringHistoImpl.cc

```diff
@@ -25,16 +25,19 @@ HGCalMulticlusteringHistoImpl::HGCalMulticlusteringHistoImpl( const edm::Paramet
 	<< "Unknown Multiclustering type '" << multiclusterAlgoType_;
     } 
 
-    edm::LogInfo("HGCalMulticlusterParameters") << "Multicluster dR: " << dr_;  
-    edm::LogInfo("HGCalMulticlusterParameters") << "Multicluster minimum transverse-momentum: " << ptC3dThreshold_;
-    edm::LogInfo("HGCalMulticlusterParameters") << "Multicluster number of R-bins for the histo algorithm: " << nBinsRHisto_<<endl;
-    edm::LogInfo("HGCalMulticlusterParameters") << "Multicluster number of Phi-bins for the histo algorithm: " << nBinsPhiHisto_<<endl;
-    edm::LogInfo("HGCalMulticlusterParameters") << "Multicluster MIPT threshold for histo threshold algorithm: " << histoThreshold_<<endl;
-    edm::LogInfo("HGCalMulticlusterParameters") << "Multicluster type of multiclustering algortihm: " << multiclusterAlgoType_;
+    edm::LogInfo("HGCalMulticlusterParameters") << "Multicluster dR: " << dr_
+    <<"\nMulticluster minimum transverse-momentum: " << ptC3dThreshold_;
+    <<"\nMulticluster number of R-bins for the histo algorithm: " << nBinsRHisto_<<endl;
+    <<"\nMulticluster number of Phi-bins for the histo algorithm: " << nBinsPhiHisto_<<endl;
+    <<"\nMulticluster MIPT threshold for histo threshold algorithm: " << histoThreshold_<<endl;
+    <<"\nMulticluster type of multiclustering algortihm: " << multiclusterAlgoType_;
+
     id_.reset( HGCalTriggerClusterIdentificationFactory::get()->create("HGCalTriggerClusterIdentificationBDT") );
     id_->initialize(conf.getParameter<edm::ParameterSet>("EGIdentification"));
-    if(multiclusterAlgoType_.find("Histo")!=std::string::npos && nBinsRHisto_!=binsSumsHisto_.size()) throw cms::Exception("Inconsistent nBins_R_histo_multicluster and binSumsHisto size in HGCalMulticlustering");
-    if(neighbour_weights_.size()!=neighbour_weights_size_) throw cms::Exception("Inconsistent size of neighbour weights vector in HGCalMulticlustering");
+    if(multiclusterAlgoType_.find("Histo")!=std::string::npos && nBinsRHisto_!=binsSumsHisto_.size()) 
+      throw cms::Exception("Inconsistent bin size") <<  "Inconsistent nBins_R_histo_multicluster ( " << nBinsRHisto_ << " ) and binSumsHisto ( " << binsSumsHisto_.size() << " ) size in HGCalMulticlustering\n");
```

Comment:
Better to use brackets `{` if the `if` statement spreads over more than one line, to avoid possible bugs in the future.